docs/supported-libraries.md

```diff
@@ -144,6 +144,7 @@ These are the supported libraries and frameworks:
 | [Vert.x Web](https://vertx.io/docs/vertx-web/java/)                                                                                         | 3.0+                          | N/A                                                                                                                                                                                                                                                                                                                                                                                     | Provides `http.route` [2]                                                                                                               |
 | [Vibur DBCP](https://www.vibur.org/)                                                                                                        | 11.0+                         | [opentelemetry-vibur-dbcp-11.0](../instrumentation/vibur-dbcp-11.0/library)                                                                                                                                                                                                                                                                                                             | [Database Pool Metrics]                                                                                                                 |
 | [ZIO](https://zio.dev/)                                                                                                                     | 2.0+                          | N/A                                                                                                                                                                                                                                                                                                                                                                                     | Context propagation                                                                                                                     |
+| [MyBatis](https://mybatis.org/mybatis-3/)                                                                                                                     | 3.2.0+                        | N/A                                                                                                                                                                                                                                                                                                                                                                                     | none                                                                                                                     |
```

Comment:
the list of libraries in alphabetical order

Reply:
I adjusted it.

Reply:
> the list of libraries in alphabetical order

I adjusted it.